docs/ref/forms/api.txt

```diff
@@ -1106,10 +1123,16 @@ they're not the only way a form object can be displayed.
 
 .. class:: BoundField
 
-   Used to display HTML or access attributes for a single field of a
-   :class:`Form` instance.
+    Used to display HTML or access attributes for a single field of a
```

Comment:
Could you please keep the original 3-space indentation? that maps to the `.. ` characters.

Reply:
This was already discussed [here](https://github.com/django/django/pull/18266#discussion_r1836612171)
I agreed this is consistent with the rest of the docs